### PR TITLE
Add snappy topic page

### LINKED ISSUE #5153: /blog: Implement snappy topic page
## Summary
Blog module 1.5.5 implements topic pages
Follow the readme at https://github.com/canonical-web-and-design/canonicalwebteam.blog to add a topic page for snapcraft.

It should be the same as https://blog.ubuntu.com/topics/snappy

### PR DESCRIPTION
**Pull request by solazio**
## Done

Add snappy topic page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/blog/topics/snappy](http://0.0.0.0:8001/blog/topics/snappy)
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Make sure it looks and behave similar to https://blog.ubuntu.com/topics/snappy

## Issue / Card

Fixes #5153

### CODE REVIEW

**Comment by solazio** [comment]
@anasereijo do you like the hero for this page? You can only see the image partially on some screen sizes...

File: templates/blog/topics/snappy.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+{% block content %} 
+
+ <section class="p-strip--accent p-blog-topic__strip is-dark is-shallow u-image-position" style="overflow: hidden; background-color: #333">
+    <div class="row header-strip u-equal-height">
+      <div class="col-6">
+        <div class="p-heading-icon">
+          <div class="p-heading-icon__header">
+            <img class="p-blog-topic__icon" src="{{ASSET_SERVER_URL}}b46c8d7d-snapcraft-logo-hor-white.svg" alt=""/>
+          </div>
+          <h2 class="p-heading--two">Package any app for any Linux and deliver updates directly</h2>
```
Comment (anthonydillon) [inline templates/blog/topics/snappy.html]: Can you make this a `p` with the heading two class. This is technically a strap line

File: static/sass/_pattern_blog-topic.scss [hunk 2]
```diff
@@ -0,0 +1,12 @@
+@mixin blog-p-topic {
+  .p-blog-topic__strip {
+    background-color: $color-accent-background;
```
Comment (anthonydillon) [inline static/sass/_pattern_blog-topic.scss]: I don't understand why we are setting this colour and then inline in the mark-up

File: static/sass/_pattern_blog-topic.scss [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  }
+
+  .p-blog-topic__icon {
```
Comment (anthonydillon) [inline static/sass/_pattern_blog-topic.scss]: Can we call this class something reuseable please. This makes this pattern unusable in another area of the site

File: static/sass/_pattern_blog-topic.scss (hunk 2 above)
Comment (solazio) [reply static/sass/_pattern_blog-topic.scss]: you are right, sorry. Initially I tried to set the colour here as a variable but I realised that for u.c the accent colour is different than blog and I had to apply the colour inline... and forgot to remove this. Will amend it.

**Comment by anasereijo** [comment]
some small changes:

- [ ] "Package any app..." to be `h3` instead of `h2`

- [ ] intro text to be `p` instead of `class="p-heading--five"`

- [ ] regarding the image they will be redesigned soon but not just now as I didn't have any design time planned for this iteration

**Comment by anasereijo** [comment]
@solazio change strip to be `is-deep`

**Pull request merged**